### PR TITLE
Pull blocklist from web

### PR DESCRIPTION
**Pull request by anishgkanchan**
BlockList.txt contains the list of blocked SSIDs. This list on getting updated is downloaded from the web and stored in the app.

### CODE REVIEW

File: src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    String getSha() {
+        if (!mIsTestMode) {
+            return "bb215d4a5e8a7fbe41cd7d61b1bb20e5745e068b";
+        }
+        String sha;
+        try {
+            //connection code
+            HttpClient client = new DefaultHttpClient();
+            HttpConnectionParams.setConnectionTimeout(client.getParams(), 10000); //Timeout Limit
+            HttpGet get = new HttpGet("https://api.github.com/repos/anishgkanchan/MozStumbler/contents/res/blocklists/BlockList.txt");
```
Comment (tamcap) [inline src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java]: Could you please edit that to reference Mozilla's repo?

File: src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    boolean getFile() {
+        Log.d("******************************************************", "@Entering getFile()");
+
+
+        boolean flag = true;
+        try {
+            //download the new file
+
+            URL url = new URL("https://raw.githubusercontent.com/anishgkanchan/MozStumbler/master/res/blocklists/BlockList.txt");
```
Comment (tamcap) [inline src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java]: And here too.

Comment (anishgkanchan) [inline src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java]: Yes sure.

File: src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java (hunk 1 above)
Comment (anishgkanchan) [inline src/org/mozilla/mozstumbler/service/blocklist/BlockListDownloader.java]: Yes sure.

**Pull request closed without merging**